projects/portal/src/app/pipes/unit-conversion.pipe.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { UnitConversionPipe } from './unit-conversion.pipe';
+
+describe('UnitConversionPipe', () => {
+  const pipe = new UnitConversionPipe();
+
+  it('transform string number to K units', () => {
+    expect(pipe.transform('1234')).toBe('1.23K');
+  });
+
+  it('transform number to M units', () => {
```

Comment:
I would like to see a more specific description of what kind of input.
For example,  `return M units when input a number with more than 7 digits`